build/purger/purger.py

```diff
@@ -326,6 +323,8 @@ def get_network_to_broadcast_rooms(api: GMatrixHttpApi) -> Dict[str, List[RoomIn
                 broadcast_room_infos.append(RoomInfo(room_id, broadcast_room_alias, server))
             except MatrixError as ex:
                 click.secho(f"Could not find room {broadcast_room_alias} with error {ex}")
+                room_id = api.create_room(alias=broadcast_room_alias, is_public=True)
+                print(room_id)
```

Comment:
`print` debugging? change this to a `log` :)

Reply:
how could this slip through? ;)